php_phongo.c

```diff
@@ -1742,22 +1856,22 @@ zend_object_iterator *php_phongo_cursor_get_iterator(zend_class_entry *ce, zval
 /* {{{ Memory allocation wrappers */
 static void* php_phongo_malloc(size_t num_bytes) /* {{{ */
 {
-	return emalloc(num_bytes);
+	return pemalloc(num_bytes, 1);
```

Comment:
AFAIK, the only benefit would be an error message from `_zend_mm_alloc_int()` when using Zend's memory manager and an OOM error is hit. Looking through the PHP 5.x code, it looks like PHP keeps heap memory in reserve to print such errors. I'd say that's preferable to the [alternatives](http://stackoverflow.com/a/6132375/162228).